packages/component-library/icons/connect.icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M14.01 5L14 1H12V5H8V1H6V5H5.99C5 4.99 4 5.99 4 6.99V12.48L7.5 16V19H12.5V16L16 12.49V6.99C16 5.99 15 4.99 14.01 5Z" fill="black"/>
```

Comment:
Could we change the `fill="black"` to `fill="#000000"` - I believe it will currently work with our process of inheriting the `color` CSS property from its parent (see below), but for the sake of continuity of our icons going forward we're trying to only use the hex value for true black.

```
// using an inline style as an example, IRL you'd probably have a className on the div and pass color through as with CSS modules

<div style="color:red">
    <Icon
      icon={connect}
      title="A descriptive title"
      role="img"
      inheritSize={true}
    />
</div>
```